test-fixtures/mssql/customers.sql

```diff
@@ -1,5 +1,5 @@
 CREATE TABLE [dbo].customers (
-  id INT IDENTITY(1,1) NOT NULL,
+  id BIGINT IDENTITY(1,1) NOT NULL,
```

Comment:
This database configuration change shows the issue and protects against a regression.